src/main/scala/com/softwaremill/realworld/users/UsersEndpoints.scala

```diff
@@ -0,0 +1,70 @@
+package com.softwaremill.realworld.users
+
+import com.softwaremill.realworld.db.{Db, DbConfig}
+import com.softwaremill.realworld.utils.*
+import io.getquill.SnakeCase
+import sttp.model.StatusCode
+import sttp.tapir.generic.auto.*
+import sttp.tapir.json.zio.jsonBody
+import sttp.tapir.server.ServerEndpoint.Full
+import sttp.tapir.ztapir.*
+import sttp.tapir.{EndpointInput, PublicEndpoint, Validator}
+import zio.json.{DeriveJsonDecoder, DeriveJsonEncoder}
+import zio.{Cause, Console, Exit, ZIO, ZLayer}
+
+import javax.sql.DataSource
+
+class UsersEndpoints(usersService: UsersService, base: BaseEndpoints):
+
+  import UsersEndpoints.given
+
+  val get: ZServerEndpoint[Any, Any] = base.secureEndpoint.get
+    .in("api" / "user")
+    .out(jsonBody[User])
+    .serverLogic(session =>
```

Comment:
so ... shouldn't this be a `.zServerLogic` as well?

Reply:
This expression:
`base.secureEndpoint.get
    .in("api" / "user")
    .out(jsonBody[User])`
returns type:
`ZPartialServerEndpoint[Any, String, UserSession, Unit, ErrorInfo, User, Any]`
and it looks in this case we don't need to specify `.zServerLogic` explicitly (`.zServerLogic` method isn't even available in this case).
This is due to the `secureEndpoint` type.

Reply:
Ah right, all clear, thanks :) We're already in ZIO-land here. Maybe we should rename the method on `ZPartialServerEndpoint` for consistency - can you create an issue in tapir?